test/dashjs.test.js

```diff
@@ -17,7 +17,12 @@
       src: 'movie.mpd',
       type: 'application/dash+xml'
     },
-    testHandleSource = function(assert, source, expectedKeySystemOptions, limitBitrateByPortal) {
+
+    testHandleSource = function(assert, source, expectedKeySystemOptions, testData) {
```

Comment:
Minor, but I think `options` or `config`, although not too descriptive themselves, might be more standard than `testData`

Reply:
`options` is already used for something else down below, for some reason I didn't think of `config`, will change.